src/ai/backend/manager/models/vfolder.py

```diff
@@ -530,13 +526,24 @@ async def prepare_vfolder_mounts(
         extra_vf_conds=extra_vf_conds,
     )
 
+    # Fast-path for empty requested mounts
+    if not accessible_vfolders:
+        return []
+    from icecream import ic
+    # add automount folder list into requested_vfolder_names
+    # and requested_vfolder_subpath
+    for vfolder in accessible_vfolders:
+        if vfolder['name'].startswith('.'):
+            requested_vfolder_names.setdefault(vfolder['name'], vfolder['name'])
+            requested_vfolder_subpaths.setdefault(vfolder['name'], '.')
+
     # for vfolder in accessible_vfolders:
     for key, vfolder_name in requested_vfolder_names.items():
         for vfolder in accessible_vfolders:
             if vfolder['name'] == requested_vfolder_names[key]:
                 break
-        else:
-            raise VFolderNotFound(f"VFolder {vfolder_name} is not found or accessible.")
+            if vfolder['name'] not in requested_vfolder_names:
+                raise VFolderNotFound(f"VFolder {vfolder_name} is not found or accessible.")
```

Comment:
```
        if vfolder_name not in [vfolder['name'] for vfolder in accessible_vfolders]:
            raise VFolderNotFound(f"VFolder {vfolder_name} is not found or accessible.")
```

how about this?
rather iterating all `accessible_vfolders`, using list comprehension seems clear

Reply:
Yes, It seems like the main purpose of the "original" commit was iterating the `name` field for accessible_vfolders.
I'll apply it.

Reply:
I think Sanghun's suggestion is good. The original inner for-loop is a boilerplate.